<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FixtureGeometryControlWidget</class>
 <widget class="QWidget" name="FixtureGeometryControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="5">
      <widget class="QLabel" name="label_17">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_16">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="originXSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
       <property name="value">
        <double>-1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Origin</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QDoubleSpinBox" name="originYSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QDoubleSpinBox" name="originZSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_15">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>OrtX</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>OrtY</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="ort1XSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
       <property name="value">
        <double>-1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QDoubleSpinBox" name="ort1YSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QDoubleSpinBox" name="ort1ZSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="ort2XSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
       <property name="value">
        <double>-1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QDoubleSpinBox" name="ort2YSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QDoubleSpinBox" name="ort2ZSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTableWidget" name="polygonWidget">
     <column>
      <property name="text">
       <string>N</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>X</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Y</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="addPushButton">
     <property name="text">
      <string>Add</string>
     </property>
     <property name="icon">
      <iconset resource="../../../resources/main.qrc">
       <normaloff>:/new/prefix1/plus.png</normaloff>:/new/prefix1/plus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="deletePushButton">
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="icon">
      <iconset resource="../../../resources/main.qrc">
       <normaloff>:/new/prefix1/minus.png</normaloff>:/new/prefix1/minus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Frame</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Polygon</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources/main.qrc"/>
 </resources>
 <connections/>
</ui>
